MAUI/Themes/HowTo.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
+To switch to light theme from dark theme, use the following code snippet.
+
+{% highlight C# %} 
+void UpdateTheme(object sender, System.EventArgs e)
+{
+    ICollection<ResourceDictionary> mergedDictionaries = Application.Current.Resources.MergedDictionaries;
+    if (mergedDictionaries != null)
+    {
+        var theme = mergedDictionaries.OfType<SyncfusionThemeResourceDictionary>().FirstOrDefault();
+        if (theme != null)
+        {
+            if (theme.SfVisualTheme is SfVisuals.DarkDefault)
+            {
+                theme.SfVisualTheme = SfVisuals.LightDefault;
+                Application.Current.UserAppTheme = AppTheme.Light;
```

Comment:
No need to add the application theme code changes

Reply:
Changes addressed.